cache-config/t3c-generate/update-to-client/update-to-client.go

```diff
@@ -32,26 +33,26 @@ import (
 
 func main() {
 	if len(os.Args) < 3 {
-		fmt.Fprintf(os.Stdout, usageStr())
+		log.Errorln(usageStr())
```

Comment:
This script hasn't initialized the loggers. You can either add a call to `log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)`, or this one's probably fine not using the loggers, either way.

This is a one-off development script, it never makes sense to run in a production system. It would be very unusual for anyone to want or need to run this thru log parsers.

Reply:
Added log.Init()